lib/src/core.dart

```diff
@@ -99,6 +111,25 @@ class AudioSession {
   Stream<void> get becomingNoisyEventStream =>
       _becomingNoisyEventSubject.stream;
 
+  /// A stream of events that occur when route change is detected (e.g. due to
+  /// unplugging the headphones or new devices detected).
+  Stream<AVAudioSessionRouteChangeReason> get routeChangeReasonStream =>
+      _routeChangeReasonSubject.stream;
+
+  /// A stream of events that occur when another application’s primary audio
+  /// has started and indicate whether optional secondary audio muting should
+  /// begin or end.
+  Stream<AVAudioSessionSilenceSecondaryAudioHintType>
+      get silenceSecondaryAudioHintStream => _silenceSecondaryAudioHintSubject.stream;
+
+  /// A stream of events when occur when a media server restarts.
+  Stream<void> get mediaServicesWereResetStream =>
+      _mediaServicesWereResetSubject.stream;
+
+  /// A stream of events when occur when a media server is lost.
+  Stream<void> get mediaServicesWereLostStream =>
+      _mediaServicesWereLostSubject.stream;
+
```

Comment:
Can also be removed, as long as they are provided in `AVAudioSession`.

Reply:
Removed!